cmd/podman/cp.go

```diff
@@ -276,3 +291,62 @@ func convertIDMap(idMaps []idtools.IDMap) (convertedIDMap []specs.LinuxIDMapping
 	}
 	return convertedIDMap
 }
+
+func streamFileToStdout(srcPath string, srcfi os.FileInfo) error {
+	file, err := os.Open(srcPath)
```

Comment:
Why do we open this here? We only use it after the IsDir() and IsArchivePath() conditions?

Reply:
I moved it after IsDir()